### DISCUSSION TITLE
Yaml file and python script that should just be copied down

### DISCUSSION THREAD

**Discussion by k8s-gpubuilder-markus** [argoproj/argo-cd#9370] [General]
Hi,

I have the situation that I have a yaml file that is for a job that gets called by a python script, providing unique values in every run. ArgoCD should not run this as it will always fail, as the yaml can't create the kubernetes object by itself. 
So I would like it to not create an object, but just copy down the yaml and .py file on change on github. Is this possible? Or should I use another tool for that? How can I exclude files to be watched by ArgoCD on github?

**Comment by rishabh625** [comment]
Hi @k8s-gpubuilder : you could specify spec.source.directory. exclude , 	// Exclude contains a glob pattern to match paths against that should be explicitly excluded from being used during manifest generation

**Reply by k8s-gpubuilder-markus** [reply]
What can I do for having these files that are excluded in one directory copied down to the kubernetes home dir when doing a sync? The python file I mean.
